security/RememberLoginHash.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+class RememberLoginHash extends DataObject {
+
+	private static $db = array (
+		'DeviceID' => 'VarChar(40)',
+		'RememberLoginHash' => 'Varchar(160)',
+	);
+
+	private static $has_one = array (
+		'Member' => 'Member',
+	);
+
+	private static $indexes = array(
+		'DeviceID' => true
+	);
+
+	/**
+	 * Determines if logging out on one device also clears existing login tokens
+	 * on all other devices owned by the member.
+	 *
+	 * @config
+	 * @var bool
+	 */
+	private static $logout_across_devices = false;
+
+	/**
+	 * Number of days the token will be valid for
+	 *
+	 * @config
+	 * @var int
+	 */
+	private static $token_expiry_days = 90;
+
+	/**
+	 * Number of days the device ID will be valid for
+	 *
+	 * @config
+	 * @var int
+	 */
+	private static $device_expiry_days = 365;
+
+	/**
+	 * If true, user can only use auto login on one device
```

Comment:
Given this is a security related setting, I'd be a bit clearer here. A user can login from multiple devices regardless of this setting, but if set to `true`, previous tokens from other devices will become invalid.